derp/derp_server.go

```diff
@@ -11,8 +11,7 @@ package derp
 import (
 	"bufio"
 	"context"
-	"crypto/rand"
-	"encoding/binary"
+	crand "crypto/rand"
```

Comment:
Why? (applies to this change in all pkgs)

If this is style we want, I'm definitely going to forget unless a linter reminds me.

Reply:
It's okay if you forget. We can lazily add it when encountered.

It's a convention I use to prevent crypto/rand vs math/rand footguns. It was in an earlier change too.